### PR TITLE
api: Cleanup sparse setup for precomputed

### PR DESCRIPTION
**Pull request by mloubout**

### CODE REVIEW

**Review by FabioLuporini — APPROVED** [review]
uncontroversial

File: docker/Dockerfile.intel [hunk 1]
```diff
@@ -117,4 +116,3 @@ FROM icx as gpu-sycl
 ENV DEVITO_ARCH="sycl"
 ENV DEVITO_LANGUAGE="sycl"
 ENV DEVITO_PLATFORM="intelgpuX"
-ENV MPICC=sycl
```
[the remark this answers is not in the document]
Comment (mloubout) [reply docker/Dockerfile.intel:120]: no intentional, leftover from previous PR

**Review by georgebisbas — APPROVED** [review]

File: examples/userapi/07_sparse_operations.ipynb
Comment (georgebisbas) [inline examples/userapi/07_sparse_operations.ipynb:1]: Why are positions differnt now?

Comment (FabioLuporini) [reply examples/userapi/07_sparse_operations.ipynb:1]: you mean in the produced output? the numerical differences?

**Pull request merged**